Actions/CalculateArtifactNames/CalculateArtifactNames.ps1

```diff
@@ -0,0 +1,41 @@
+Param(
+    [Parameter(HelpMessage = "Settings from repository in compressed Json format", Mandatory = $true)]
+    [string] $settingsJson,
+    [Parameter(HelpMessage = "Name of the built project", Mandatory = $true)]
+    [string] $project,
+    [Parameter(HelpMessage = "Buildmode used when building the artifacts", Mandatory = $true)]
+    [string] $buildMode,
+    [Parameter(HelpMessage = "Name of the branch the workflow is running on", Mandatory = $true)]
+    [string] $branchName
+)
+
+function Set-EnvVariable([string] $name, [string] $value) {
+    Write-Host "Assigning $value to $name"
+    Add-Content -Path $env:GITHUB_OUTPUT -Value "$name=$value"
```

Comment:
I thought that if your action doesn't have any outputs defined in action.yaml, I don't think outputting anything to GITHUB_OUTPUT will have any meaning.
I saw that you defined outputs in CI/CD grabbing values from this action - does that really work?

Reply:
You're right :( I discovered this yesterday as well. I can define the outputs for CalculateArtifactNames in the action.yaml but for DetermineBuildOrder this might not be possible, since our outputs names are variable e.g. projects1json, projects2json

Reply:
and here you need to add the outputs to the test as well (see other actions with output)

Reply:
Outputs added to CalculateArtifactNames